contrib/experimental/setup.py

```diff
@@ -30,7 +30,6 @@
         "Topic :: Software Development :: Testing",
         "License :: OSI Approved :: Apache Software License",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
```

Comment:
Since you are bothering to touch this file, you want to add a 3.9 line?

Reply:
Or maybe leave it until we "officially" support 3.10 and are adding a 3.10 line to the main setup.py

Reply:
I'll add 3.9 but wait on 3.10 since we actually get some failures when building on that env